packages/react-core/src/components/List/ListItem.tsx

```diff
@@ -16,7 +16,7 @@ export const ListItem: React.FunctionComponent<ListItemProps> = ({
 }: ListItemProps) => (
   <li className={css(icon && styles.listItem)} {...props}>
     {icon && <span className={css(styles.listItemIcon)}>{icon}</span>}
-    {children}
+    <span className={css('pf-v5-c-list__item')}>{children}</span>
```

Comment:
Could we avoid hard coding the classname here? use the object from react-styles instead?

Reply:
There is no object entry for this class is the issue, because it's not applying any styles currently. I can remove it?

Reply:
Gotcha updated! Also caught upon double checking core that the class only gets applied if an icon is present so updated the logic/snapshots too.